modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported log data output types
+
+[role="_abstract"]
```

Comment:
Hi @rolfedh - we are not using these Pv2 tags at the moment. Do you mind removing this as a follow up PR?

See my [note](https://coreos.slack.com/archives/C2ZA5QGMV/p1612810770135000?thread_ts=1612790580.125900&cid=C2ZA5QGMV) in Slack.

Reply:
@vikram-redhat Sure - no problem. I'll fix it today. This particular case comes from the redhat-documentation/modular-docs templates. We should have a tete-a-tete with Emily Murphy to avoid expanding issues like this.

Reply:
I've removed  [role="_abstract"] in https://github.com/openshift/openshift-docs/pull/30088, which is WIP. Please note that the [role=*] attribute also appears in the following locations"
```
Results for "[role=" in "*.adoc"

/home/rdlugyhe/openshift-docs/migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc:
  62:  [role="_additional-resources"]


/home/rdlugyhe/openshift-docs/migration/migrating_3_4/troubleshooting-3-4.adoc:
  38:  [role="_additional-resources"]


/home/rdlugyhe/openshift-docs/migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc:
  61:  [role="_additional-resources"]


/home/rdlugyhe/openshift-docs/migration/migrating_4_1_4/troubleshooting-4-1-4.adoc:
  37:  [role="_additional-resources"]


/home/rdlugyhe/openshift-docs/migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc:
  61:  [role="_additional-resources"]


/home/rdlugyhe/openshift-docs/migration/migrating_4_2_4/troubleshooting-4-2-4.adoc:
  37:  [role="_additional-resources"]
```